src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -375,6 +375,10 @@ private function addHttpCacheSection(ArrayNodeDefinition $rootNode): void
                                     ->prototype('scalar')->end()
                                     ->info('URLs of the Varnish servers to purge using cache tags when a resource is updated.')
                                 ->end()
+                                ->integerNode('max_header_length')
+                                    ->defaultValue(7500)
```

Comment:
How is this default derived?

Reply:
Default Apache max headers size is 8K so the idea is to let some room for other headers, hence the 7500 value